### ISSUE TITLE
URGENT - error android api level 29

### ISSUE THREAD

**Issue by hvar90** [WindSekirun/MediaResizer#11]
one user from my app send me this error 

null cannot be cast to non-null type  android.graphics.Bitmap

he has android 10 apii level 29

the error happens when i use your library 
but i only see this error with android 10 api level 29

```
ImageResizeOption resizeOption = new ImageResizeOption.Builder()
                    .setImageProcessMode(ImageMode.ResizeAndCompress)
                    .setImageResolution(width, height)
                    .setBitmapFilter(false)
                    .setCompressQuality(75)
                    .setCompressFormat(Bitmap.CompressFormat.JPEG)
                    .setScanRequest(ScanRequest.TRUE)
                    .build();

ResizeOption option = new ResizeOption.Builder()
                    .setMediaType(MediaType.IMAGE)
                    .setImageResizeOption(resizeOption)
                    .setTargetPath(inputF.getAbsolutePath())
                    .setOutputPath(outputF.getAbsolutePath())
                    .setCallback((code, output) -> {
                        if (code > 0)
                         some code
                        else
                         some code
                    }).build();

MediaResizer.process(option);

```
maybe kotlin code is necessary to change 
i found this error 
https://stackoverflow.com/questions/55967515/kotlin-typecastexception-null-cannot-be-cast-to-non-null-type-android-graphics
i dont know if it has relation with this problem     

**update**:

i use the emulator of android studio and i checked the problem happens with android 10
this is the log of the error 

```
Caused by: kotlin.TypeCastException: null cannot be cast to non-null type android.graphics.Bitmap
        at pyxis.uzuki.live.mediaresizer.processor.ImageProcessorKt.resizeImageInternally(ImageProcessor.kt:36)
        at pyxis.uzuki.live.mediaresizer.processor.ImageProcessorKt.resizeImage(ImageProcessor.kt:24)
        at pyxis.uzuki.live.mediaresizer.MediaResizer.process(MediaResizer.kt:24)
        at schan.main.ChatActivity.processImage(ChatActivity.java:1943)
```

     

thanks for any help you can give me

**Comment by hvar90** [comment]
hi again

this bug will be resolve in the future ? 
thanks for the answer

**Comment by hvar90** [comment]
more errors in android 10 

`ava.lang.RuntimeException: 
  at android.app.ActivityThread.deliverResults (ActivityThread.java:5096)
  at android.app.ActivityThread.handleSendResult (ActivityThread.java:5137)
  at android.app.servertransaction.ActivityResultItem.execute (ActivityResultItem.java:51)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2146)
  at android.os.Handler.dispatchMessage (Handler.java:107)
  at android.os.Looper.loop (Looper.java:237)
  at android.app.ActivityThread.main (ActivityThread.java:7762)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1047)


Caused by: kotlin.TypeCastException: 
  at pyxis.uzuki.live.mediaresizer.processor.ImageProcessorKt.resizeImageInternally (ImageProcessor.kt:36)
  at pyxis.uzuki.live.mediaresizer.processor.ImageProcessorKt.resizeImage (ImageProcessor.kt:24)
  at pyxis.uzuki.live.mediaresizer.MediaResizer.process (MediaResizer.kt:24)
  at schan.main.ChatActivity.resizeWallpaper (ChatActivity.java:2209)
  at schan.main.ChatActivity.onActivityResult (ChatActivity.java:2227)
  at android.app.Activity.dispatchActivityResult (Activity.java:8318)
  at android.app.ActivityThread.deliverResults (ActivityThread.java:5089)`

**Comment by WindSekirun** [comment]
I can't reproduce this issue on my Android 10 device (S10+). However, it is unreasonable to crash when Bitmap is null. #12 will fix crash when bitmap is null (TypeCastException).

Also, This library uses [BitmapFactory.decodeFile](https://developer.android.com/reference/android/graphics/BitmapFactory#decodeFile(java.lang.String,%20android.graphics.BitmapFactory.Options)) to convert File to Bitmap object.

It's probably caused when file is empty or you enter a file that Android doesn't support. 

I published 1.1.1 version with #12 , It can be used in 1 hours. Try that version and reopen issue when this issue is reproduced.

**Comment by hvar90** [comment]
> I can't reproduce this issue on my Android 10 device (S10+). However, it is not unreasonable to crash when Bitmap is null. #12 will fix crash when bitmap is null (TypeCastException).
> 
> Also, This library uses [BitmapFactory.decodeFile](https://developer.android.com/reference/android/graphics/BitmapFactory#decodeFile(java.lang.String,%20android.graphics.BitmapFactory.Options)) to convert File to Bitmap object.
> 
> It's probably caused when file is empty or you enter a file that Android doesn't support.
> 
> I published 1.1.1 version with #12 , It can be used in 1 hours. Try that version and reopen issue when this issue is reproduced.

thanks ...
i will test with the new version

**Comment by hvar90** [comment]
the problem with android 10 was other 
here is explained the problem 
https://medium.com/@sriramaripirala/android-10-open-failed-eacces-permission-denied-da8b630a89df

thats the reason the file was empty

but anyway thanks for fixing the crash
